src/select-menu/docs/examples/SelectMenu-multi.example

```diff
@@ -0,0 +1,46 @@
+<Manager>
+       {({ setState, state }) => (
+              <SelectMenu
+                title="Select name"
+                options={options}
+                selected={state.selected}
+                onSelect={item => {
+            let selected
+            if (Array.isArray(state.selected)) {
+              selected = [...state.selected, item.value]
+            } else {
+              selected = arrify(item.value)
+            }
+            setState({
+              selected,
+              selectedNames: selected.toString().substr(0, 25) + '...'
+            })
+          }}
+          onDeselect={item => {
+            const DeselectedItemIndex = state.selected.indexOf(item.value)
+            const selectedItems = state.selected.filter(
+              (_item, i) => i !== DeselectedItemIndex
+            )
+
+            const selectedItemsLength = selectedItems.length
+            let selectedNames = ''
+            switch (true) {
+              case selectedItemsLength === 0:
+                selectedNames = ''
+                break
+              case selectedItemsLength === 1:
+                selectedNames = selectedItems.toString()
+                break
+              case selectedItemsLength > 1:
+                selectedNames = selectedItems.toString().substr(0, 25) + '...'
```

Comment:
This is cool and a nice solution to this problem! Alternatively, I could see simply `2-n items` as the label.

Reply:
Exactly the wisdom I was looking for!